### PR TITLE
2.3.x

### PR DESCRIPTION
**Pull request by tahmmee**

### CODE REVIEW

File: charts/couchbase-operator/README.md [hunk 1]
```diff
@@ -38,24 +38,23 @@ for more information about customizing and managing your charts.
...
-| admissionController.disableValidatingWebhook | bool | `false` | Disable Validationg Webhook to skip validation of Couchbase cluster creation |
-| admissionController.image.repository | string | `"couchbase/admission-controller"` |  |
-| admissionController.image.tag | string | `"2.3.0"` |  |
-| admissionController.imagePullPolicy | string | `"IfNotPresent"` |  |
-| admissionController.imagePullSecrets | list | `[]` | imagePullSecrets is an optional list of references to secrets to use for pulling images |
+| admissionController.commandArgs | object | `{"default-file-system-group":true,"validate-secrets":true,"validate-storage-classes":true}` | Set of command-line flags to pass on to the Admission Controller to modify its behavior. Do not change. |
+| admissionController.disableValidatingWebhook | bool | `false` | Disable the creation of Validationg Webhook. Setting to 'false' may be helpful when installing into a restricted environment (ie STRICT MODE ISTIO), since disabling avoids performing resource fetching and validation from the kube api. |
```
Comment (patrick-stephens) [inline charts/couchbase-operator/README.md:42]: Typo on validating, also maybe reword for strict mTLS for other service meshes

File: charts/couchbase-operator/README.md.adoc [hunk 2]
```diff
@@ -213,60 +212,60 @@
...
-| couchbaseOperator.commandArgs.pod-create-timeout | string | `"10m"` | pod creation timeout 
-| couchbaseOperator.image | object | `{"repository":"couchbase/operator","tag":"2.3.0"}` | image config 
-| couchbaseOperator.imagePullPolicy | string | `"IfNotPresent"` |  
-| couchbaseOperator.imagePullSecrets | list | `[]` | imagePullSecrets is an optional list of references to secrets  to use for pulling images 
-| couchbaseOperator.name | string | `"couchbase-operator"` | name of the couchbase operator 
-| couchbaseOperator.nodeSelector | object | `{}` | nodeSelector for couchbase-operator pod assignment -- Ref: https://kubernetes.io/docs/user-guide/node-selection/ 
-| couchbaseOperator.resources | object | `{}` | resources of couchbase-operator 
+| collectiongroups | object | `{}` | Uncomment to create a "couchbasecollectiongroups" resource Defines a group of collections. A collection is a data container, defined on Couchbase Server, within a bucket whose type is either Couchbase or Ephemeral. See xref:resource/couchbasecollectiongroup.adoc[`CouchbaseCollectionGroup` resource reference]
```
Comment (patrick-stephens) [inline charts/couchbase-operator/README.md.adoc:215]: Do xrefs work outside of repo?

File: charts/couchbase-operator/templates/_helpers.tpl [hunk 3]
```diff
@@ -414,7 +414,7 @@ Generate name of service account to use for backups
 */}}
 {{- define "couchbase-cluster.backup.service-account" -}}
 {{- $clusterName := (include "couchbase-cluster.clustername" .) -}}
-{{- default (printf "backup-%s" $clusterName) .Values.cluster.serviceAccountName -}}
+{{- default (printf "backup-%s" $clusterName) .Values.cluster.backup.serviceAccountName -}}
```
Comment (patrick-stephens) [inline charts/couchbase-operator/templates/_helpers.tpl:417]: Need to merge to main as well

File: charts/couchbase-operator/values-all.yaml [hunk 4]
```diff
@@ -42,39 +50,51 @@ couchbaseOperator:
...
   tolerations: []
-  # -- turn on verbose logging
+  # -- Determines whether the admission controller should log all of its validation notices within the console.
+  # When set to false, only validation errors are logged within the pod’s console.
   verboseLogging: false
-  # -- Run as non-root container
+  # -- Specify whether to run as a non-root user. Running as non-root ensures least privilege.
   runAsNonRoot: true
-  # -- Disable Validationg Webhook to skip validation of Couchbase cluster creation
+  # -- Disable the creation of Validationg Webhook. Setting to 'false' may be helpful when
```
Comment (patrick-stephens) [inline charts/couchbase-operator/values-all.yaml:83]: Typo

File: charts/couchbase-operator/values-all.yaml [hunk 5]
```diff
@@ -42,39 +50,51 @@ couchbaseOperator:
...
+  # installing into a restricted environment (ie STRICT MODE ISTIO), since disabling
```
Comment (patrick-stephens) [inline charts/couchbase-operator/values-all.yaml:84]: Strict mTLS

File: charts/couchbase-operator/values-all.yaml [hunk 6]
```diff
@@ -194,6 +217,8 @@ collectiongroups: {}
 #   names: []
 
 # --  Uncomment to create a "couchbasecollections" resource
+# A collection is a data container, defined on Couchbase Server, within a bucket whose type is either Couchbase or Ephemeral.
+# See xref:resource/couchbasecollection.adoc[`CouchbaseCollection` resource reference]
```
Comment (patrick-stephens) [inline charts/couchbase-operator/values-all.yaml:221]: Xref?

File: tools/value-generation/gen.py [hunk 7]
```diff
@@ -151,6 +151,7 @@ def postProcessCluster(crd_value, value_map, comment_map) :
 
   value_map[crd_value]['buckets']['managed'] = True
   value_map[crd_value]['image'] = 'couchbase/server:6.6.2'
```
Comment (patrick-stephens) [inline tools/value-generation/gen.py:153]: Probably should change this

**Review by patrick-stephens — COMMENTED** [review]
Do the xrefs work? 
We need to merge that backup change as well to a 2.2.1 release.

File: charts/couchbase-operator/templates/_helpers.tpl (hunk 3 above)
Comment (tahmmee) [reply charts/couchbase-operator/templates/_helpers.tpl:417]: Looks like you've got it!

File: charts/couchbase-operator/README.md.adoc (hunk 2 above)
Comment (tahmmee) [reply charts/couchbase-operator/README.md.adoc]: nope. fixed
